source/docs/software/sensors/analog-potentiometers-software.rst

```diff
@@ -1,13 +1,11 @@
-.. _analog-potentiometers-software:
-
 Analog Potentiometers - Software
 ================================
 
-.. note:: This section covers analog potentiometers in software.  For a hardware guide to analog potentiometers, see :ref:`analog-potentiometers-hardware`.
+.. note:: This section covers analog potentiometers in software.  For a hardware guide to analog potentiometers, see :ref:`docs/hardware/sensors/analog-potentiometers-hardware:Analog Potentiometers - Hardware`.
```

Comment:
This is pretty verbose... pity there's not a shorter way to do this?

Reply:
`doc` I wish would, but unfortunately it only works in current directory and subdirectories, not based on configuration root ☹️